pkgs/servers/unifi/default.nix

```diff
@@ -35,7 +35,7 @@ let
       description = "Controller for Ubiquiti UniFi access points";
       license = licenses.unfree;
       platforms = platforms.unix;
-      maintainers = with maintainers; [ erictapen ];
+      maintainers = with maintainers; [ bachp 0x4A6F ];
```

Comment:
This should be [ ... maintainers."0x4A6F" ]

Reply:
I don't think so. There is a `with maintainers` in front.

Reply:
Fixed